tests/helpers.py

```diff
@@ -21,3 +26,39 @@ def create_unique_resource_name(resource_description, resource_type):
         datetime.datetime.now().strftime("%d%H%M%S%f")
     )
     return f"{resource_type}_{resource_description[:23]}_{current_date_time[:10]}"
+
+
+def create_ocs_obj(**kwargs):
+    """
+    Create an instance of BaseOCSClass
+
+    Args:
+        kwargs (dict): Dictionary of the OCS resource
+
+    Returns:
+        BaseOCSClass: An instance of BaseOCSClass
+
+    """
+    return BaseOCSClass(**kwargs)
```

Comment:
why do we need this,  one can directly call OCS()

Reply:
done